cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofileparser.json

```diff
@@ -0,0 +1,23 @@
+{
+  "office-id": "SWT",
+  "location-id": "TIMESERIESPROFILE_LOC",
```

Comment:
If I remember correctly a base location has a max length of 24 characters. (I didn't count, it just looks long.)

Reply:
picked a shorter name